md/release-notes.md

```diff
@@ -8,17 +8,17 @@ Simply Bonita.
 
 <a id="living-application-development-and-deployment"/>
 
-### Industrialization of Living Applications in Bonita Studio: Profiles
-Our goal is to provide you with as few deployment artifacts as possible when creating applications.
-To achieve this, we need all application resources to be available in the Studio / UI Designer development suite.
-After application descriptors, profiles are now available in Bonita Studio.
-In Teamwork editions, you can edit the default profiles mapping with organization entities.
-In Efficiency and Performance editions, you can also create custom profiles, map them to the organization, and create their Bonita Portal menus if needed.
+### [Industrialization of Living Applications in Bonita Studio: Profiles](profileCreation.md)
+Our goal is to provide you with as few deployment artifacts as possible when creating applications.  
+To achieve this, we need all application resources to be available in the Studio / UI Designer development suite.  
+After application descriptors, profiles are now available in Bonita Studio.  
```

Comment:
I suggest to put link on word [profiles] and not on section title...what did you tkink about this ?

Reply:
ok for me